grow/templates/jinja_dependency.py

```diff
@@ -0,0 +1,112 @@
+"""Track the jinja2 rendering dependencies."""
```

Comment:
Good point. I've added in some better explanation of why I am using that method and how to go about updating when updating jinja.